lib/domain/model/session.dart

```diff
@@ -166,6 +176,165 @@ class Credentials {
         PreciseDateTime.parse(data['refresh']['expireAt']),
       ),
       UserId(data['userId']),
+      MyUser(
+        id: data['user']['id'],
+        num: UserNum(data['user']['num']),
+        login: UserLogin(data['user']['login']),
+        emails: MyUserEmails(
+          confirmed: (data['user']['emails']['confirmed'] as List<String>)
```

Comment:
Why the JSON mapping logic of `MyUser` is contained in the `Credentials` model? It can and thus must be encapsulated within `MyUser`. However, imagine changing a type of some field of the `MyUser` model. This will surely break stuff and won't be noticed during compile time. Runtime errors are the ones most hard to debug. The conversion above is allowed there, as it contain easily maintainable amount of code, and the logic is kinda encapsulated.